service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleIntakeMessageServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.mskcc.cmo.metadb.service.impl;
+
+import org.mskcc.cmo.messaging.Gateway;
+import org.mskcc.cmo.messaging.MessageConsumer;
+import org.mskcc.cmo.messaging.model.SampleMetadataEntity;
+import org.mskcc.cmo.metadb.service.SampleIntakeMessageService;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ *
+ * @author ochoaa
+ */
+@Service
+public class SampleIntakeMessageServiceImpl implements SampleIntakeMessageService, MessageConsumer {
+    private final String NEW_SAMPLE_INTAKE = "NEW_SAMPLE_INTAKE";
+    private final String NEW_SAMPLE_SUBMISSION = "NEW_SAMPLE_SUBMISSION";
+
+    private final Log LOG = LogFactory.getLog(SampleIntakeMessageServiceImpl.class);
+
+    @Autowired
+    private Gateway messagingGateway;
+
+    @Override
+    public void onSampleIntakeMessage() throws Exception {
```

Comment:
From where and when does this get called?

Reply:
this is more temp code put in place where sample intake handling would eventually go